src/client/pps/pps.proto

```diff
@@ -313,6 +313,7 @@ enum PipelineState {
 // by pointing to a PFS commit, de facto tracks the pipeline's version)
 message EtcdPipelineInfo {
   PipelineState state = 1;
+  bool stopped = 6;
```

Comment:
I think it makes more sense to store this field in the spec commit rather than Etcd. In my mind the EtcdPipelineInfo is for the "real world state" of the pipeline rather than the users intent. That's one of the nice benefits of this PR in my mind, it lets us move `stopped` away from the volatile state into the spec commit.